spec/features/users/stream_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'spec_helper'
+
+feature 'User Stream Information' do
+  let(:password) { "foobar" }
+  let!(:user) { create :user, raw_password: password }
+
+  feature 'stream administration' do
+    scenario 'user updates their stream' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.